thrust/examples/padded_grid_reduction.cu

```diff
@@ -82,9 +80,9 @@ int main()
   thrust::device_vector<float> data(M * N, -1);
 
   // initialize valid values in grid
-  for (int i = 0; i < M; i++)
+  for (size_t i = 0; i < static_cast<size_t>(M); i++)
   {
-    for (int j = 0; j < n; j++)
+    for (size_t j = 0; j < static_cast<size_t>(n); j++)
```

Comment:
Suggestion: I would stay with `int`. I don't see the benefit of using `size_t` here.